bridge/Dockerfile

```diff
@@ -13,5 +15,7 @@ RUN pip install -e .
 RUN chown -R 1000:1000 /app
 USER 1000
 
+ENV MIXPANEL_API_KEY=${MIXPANEL_KEY}
```

Comment:
Annoyingly you can't use the same name for an ARG and ENV unless you're using buildkit. Don't want to rely on this so...